playbooks/deploy_cluster.yml

```diff
@@ -5,8 +5,8 @@
 
 - import_playbook: common/private/control_plane.yml
 
-- import_playbook: openshift-node/private/join.yml
-
 - import_playbook: openshift-node/private/config.yml
 
+- import_playbook: openshift-node/private/join.yml
```

Comment:
This is not correct.  We want to join nodes before we add control plane components.